src/drive/targets/mobile/config.xml

```diff
@@ -17,6 +17,7 @@
         <preference name="android-minSdkVersion" value="21" />
         <preference name="StatusBarBackgroundColor" value="#D6D8DA" />
         <preference name="AndroidPersistentFileLocation" value="Compatibility" />
+        <preference name="AndroidLaunchMode" value="singleTask" />
```

Comment:
Is there a chance this breaks something? Like opening the app from banks?

Reply:
Nop. I have added a description for this specific commit.

https://github.com/cozy/cozy-drive/pull/1918/commits/80f0aa9cbcb5d163bb9f12bb54a7406fa0459588